src/GoodsList.tsx

```diff
@@ -1,14 +1,21 @@
 import React from 'react';
+import 'bulma/css/bulma.min.css';
```

Comment:
It's better to import CSS in a main/root file (e.g., index.tsx or App.tsx) instead of component files. 📦